src/main/java/creativei/entity/Inquiry.java

```diff
@@ -19,12 +23,22 @@
         @UniqueConstraint(columnNames = "alternate_phone")
 })
 
-public class Inquiry extends BaseEntity implements Serializable {
+public class Inquiry  extends BaseEntity implements Serializable  {
+
+    public Inquiry(InquiryVo inquiryVo) throws Exception {
+        this.name=inquiryVo.getName();
+        this.areaOfInterest=inquiryVo.getAreaOfInterest();
+        this.phoneNumber=inquiryVo.getMobile();
+        this.email=inquiryVo.getEmail();
+        this.highestEducation=inquiryVo.gethQualification();
+        this.dob= LocalizationUtil.stringToDateConverter(inquiryVo.getDob());
+        this.gender=inquiryVo.getGender();
```

Comment:
value in vo should be of string type. This can be in a different case as well. Add a method in your enums to get enum value from string.
Value of enum in vo classes should be string.